Dockerfile

```diff
@@ -1,53 +1,50 @@
-FROM rocker/r-ver:4.3.1
+FROM rocker/r-ver:4.3.2
+
+# Set the working directory to setup. Uses a dedicated directory instead of
+# root since otherwise renv will try to scan every subdirectory
+WORKDIR /setup
```

Comment:
Slight re-configuration of the Dockerfile here. `renv` was stupidly scanning the entire filesystem when using the root directory and thus timing out. I created a dedicated working directory just for setup stuff, sort of following the multi-stage build pattern.